kubetest/main.go

```diff
@@ -94,12 +96,14 @@ func defineFlags() *options {
 	flag.StringVar(&o.deployment, "deployment", "bash", "Choices: none/bash/kops/kubernetes-anywhere")
 	flag.BoolVar(&o.down, "down", false, "If true, tear down the cluster before exiting.")
 	flag.StringVar(&o.dump, "dump", "", "If set, dump cluster logs to this location on test or cluster-up failure")
+	flag.BoolVar(&o.enableLogexporter, "enable-logexporter", false, "If true, use logexporter tool to upload logs from nodes to GCS directly (saves time and is recommended)")
```

Comment:
`--enable-logexporter=gs://foo/bar` or something like that seems cleaner and less redundant than `--enable-logexporter --gcs-artifacts-dir=gs://foo/bar`